src/cogs/twitch/arg_resolvers/user_id_resolver.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional
+
+# from .. import BroadcasterService  # This import breaks everything, which is why it's commented out.
```

Comment:
you can never import from the root __init__.py file pretty much. you have to import it from the actual source file. Something like: `from ..services.broadcaster_service` import BroadcasterService` should be valid and not cause circular imports